### PR TITLE
KEYCLOAK-1542 more informations shown on Server Info page

### PR DESCRIPTION
**Pull request by velias**
replaces PR #1478

### CODE REVIEW

File: forms/common-themes/src/main/resources/theme/base/admin/resources/partials/server-info.html [hunk 1]
```diff
@@ -1,16 +1,113 @@
...
+        <tr>
+            <td>OS Architecture</td>
+            <td>{{serverInfoPage.systemInfo.osArchitecture}}</td>
+        </tr>
+        </table>
+        </div>
+    </fieldset>    
+
+   	
+    <fieldset ng-show="serverInfoPage.jpaInfo">
```
Comment (stianst) [inline forms/common-themes/src/main/resources/theme/base/admin/resources/partials/server-info.html]: Looks like this left in by mistake?

**Comment by stianst** [comment]
Looks good, I'm going to refactor it a bit today though. The server-info as already a bit messy and with the addition of more stuff it's even messier. Not sure I like the addition of a completely different endpoint/resource though. Firstly the endpoints shouldn't be linked to admin console (so server info page is not a good name), they should be generic rest endpoints available to anyone. Further it ends up with a fair bit of code duplication.

**Comment by stianst** [comment]
Actually, I think I might just add it to the server info endpoint, but make sure the admin console caches it so it only request it once, except it'll get refreshed when viewing the server info page.

**Comment by velias** [comment]
Template is patched now.
I leave decision about endpoints on you, it is always problem to design it correctly and you have better understanding of whole KC REST API.
Current overlap in these two endpoints is only version, server time and provider structure, but even it contains some additional info. Client side caching may be a problem as endpoints contain config data which are changed in console itself, bad invalidation may bring errors into console (try to imagine that two admins do some configuration changes at same time).
Maybe we should rename /serverinfopage endpoint to something not so closely related to the page itself, but describing nature/purpose of info, eg /systemoperationalinfo

**Comment by stianst** [comment]
Closing this as I'm preparing a new PR with some additional changes

**Pull request closed without merging**